service/handlers.go

```diff
@@ -0,0 +1,152 @@
+package service
+
+import (
+	"fmt"
+	"github.com/avigailbr/book_service/datastore"
+	"github.com/avigailbr/book_service/models"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+type UpdateBook struct {
+	Id    string `json:"id" binding:"required`
+	Title string `json:"title" binding:"required"`
+}
+
+func Ping(c *gin.Context) {
+	c.JSON(http.StatusOK, gin.H{
+		"message": "pong",
+	})
+}
+
+func BookInfo(c *gin.Context) {
+	id := c.Param("id")
+	db := c.MustGet("DB").(datastore.IBookStorer)
```

Comment:
I'm not familiar with this syntax but if it works I guess it's ok.
you can always use a reg var as well :)